tiny_dnn/models/alexnet.h

```diff
@@ -19,14 +19,21 @@ namespace models {
 class alexnet : public network<sequential> {
  public:
   explicit alexnet(const std::string &name = "") : network<sequential>(name) {
-    *this << conv<relu>(224, 224, 11, 11, 3, 64, padding::valid, true, 4, 4);
-    *this << max_pool<identity>(54, 54, 64, 2);
-    *this << conv<relu>(27, 27, 5, 5, 64, 192, padding::valid, true, 1, 1);
-    *this << max_pool<identity>(23, 23, 192, 1);
-    *this << conv<relu>(23, 23, 3, 3, 192, 384, padding::valid, true, 1, 1);
-    *this << conv<relu>(21, 21, 3, 3, 384, 256, padding::valid, true, 1, 1);
-    *this << conv<relu>(19, 19, 3, 3, 256, 256, padding::valid, true, 1, 1);
-    *this << max_pool<identity>(17, 17, 256, 1);
+    // todo: (karandesai) shift this to tiny_dnn::activation
+    using relu = relu_layer;
```

Comment:
Is this comment still valid?
https://github.com/tiny-dnn/tiny-dnn/pull/651/files#diff-308324af2ff351c1b1ceaf7d80e862b4R116